pkg/networkserver/grpc_deviceregistry_test.go

```diff
@@ -309,7 +309,7 @@ func TestDeviceRegistrySet(t *testing.T) {
 						JoinEUI:                &types.EUI64{0x42, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00},
 						DevEUI:                 &types.EUI64{0x42, 0x42, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00},
 					},
-					FrequencyPlanID:   test.EUFrequencyPlanID,
+					FrequencyPlanID:   "",
```

Comment:
No need to set zero value

Reply:
The test was testing if some field was missing. That check is removed, so now we empty a field that actually makes the request invalid (i.e. what the test is testing)

Reply:
I understand, but that is not relevant here - this field is `""` by default, no need to set it explicitly

Reply:
- The point of this test is creating an invalid end device
- For the future developer (including yourself), let's make it clear what is invalid about it, i.e. setting an empty field explicitly
- This is reverted anyway because what's now invalid is the empty `DevAddr` (still explicitly set)